src/components/UiTooltip/TooltipWrapper.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  ClickAwayListener,
+  Tooltip,
+  Typography,
+  useMediaQuery,
+} from '@mui/material';
+import React from 'react';
+
+import { UiTooltipProps } from './types';
+
+export default function WrapperUiTooltip({
+  children,
+  title,
+  placement,
+  arrow,
+  sx,
+}: UiTooltipProps): React.ReactElement {
+  const [open, setOpen] = React.useState(false);
+  const isWideScreenMaxWidth: boolean = useMediaQuery('(max-width: 640px)');
+  const isWideScreenMinWidth: boolean = useMediaQuery('(min-width: 640px)');
+
+  React.useEffect(() => {
+    if (isWideScreenMaxWidth || isWideScreenMinWidth) {
+      setOpen(false);
+    }
+  }, [isWideScreenMaxWidth, isWideScreenMinWidth]);
+
+  function handleTooltipClose(): void {
+    setOpen(false);
+  }
+  function handleTooltipToogle(): void {
+    setOpen(!open);
+  }
+  return (
+    <ClickAwayListener onClickAway={() => handleTooltipClose()}>
+      <Tooltip
+        open={open}
+        title={title}
+        placement={placement}
+        arrow={arrow}
+        sx={sx}
+      >
+        <Typography component="span" onClick={() => handleTooltipToogle()}>
```

Comment:
same

Reply:
![image](https://github.com/VilnaCRM-Org/website/assets/112288735/b60ea345-e193-4125-8374-0265a8a9a806)
We have eslint rule for this

Reply:
hmm, okay

Reply:
but i think this is not good idea to write code like this, I think we need to remove this rule, because providing just function in some cases mo useful and looks more clear